app/build.gradle.kts

```diff
@@ -93,6 +93,9 @@ dependencies {
     implementation(libs.room.ktx)
 
     implementation(libs.date.time.picker)
+
+    implementation("androidx.compose.material:material:1.6.7") // TODO
```

Comment:
We have material3 library installed already.. consider using those components instead?